Cargo.toml

```diff
@@ -85,4 +85,4 @@ codegen-units = 1
 [patch.crates-io]
 # TODO: Remove once chacha20poly1305 0.10 appears in libp2p's dependencies
 chacha20poly1305 = { git = "https://github.com/RustCrypto/AEADs", rev = "06dbfb5571687fd1bbe9d3c9b2193a1ba17f8e99" }
-libp2p = { git = "https://github.com/subspace/rust-libp2p", branch = "subspace-v2" }
+libp2p = { git = "https://github.com/subspace/rust-libp2p", branch = "subspace-v3" }
```

Comment:
why not a revision instead of branch?

Reply:
Whoever added it did it this way, so I keep doing it too, really no specific reason :shrug: